book/internal-services.md

```diff
@@ -0,0 +1,83 @@
+**Internal, Private Cluster** - A cluster that is not externally addressable but which has access to Red Hat's network.
+**App Interface** - A repository that serves as a central coordination point for hosted services operated by the Application SRE team.
+
+## Resources
+Below is the list of CRs that the Internal Service is responsible for interacting with:
+
+### CREATE
+
+| Custom Resource             | When?                                                                                                             | Why?                                           |
+|-----------------------------|-------------------------------------------------------------------------------------------------------------------|------------------------------------------------|
+| PipelineRun                 | Once a InternalRequest has been reconciled and the Pipeline to run has been discovered and is ready for execution | To perform the steps in the Internal Pipeline  |
+
+### READ
+
+| Custom Resources     | When?                                                       | Why?                                                                   |
+|----------------------|-------------------------------------------------------------|------------------------------------------------------------------------|
+| Config               | During controller startup and during each reconcile attempt | To obtain configuration information                                    |
+
+### UPDATE
+
+| Custom Resource  | When?                                                              | Why?                                                                      |
+|------------------|--------------------------------------------------------------------|---------------------------------------------------------------------------|
+| InternalRequest  | During the lifecycle of an attempt to complete an InternalRequest  | To provide status for the execution of the Pipeline to the remote cluster |
+
+### WATCH
+
+| Custom Resource            | When?                                                      | Why?                                                                               |
+|----------------------------|------------------------------------------------------------|------------------------------------------------------------------------------------|
+| InternalRequest            | Always                                                     | To provide an API to process an internal request                                   |
+| PipelineRun                | Once the PipelineRun is created                            | To relay the Internal PipelineRun status to the remote InternalRequest for viewing |
+
+## Security Risk Mitigations
+
+Enabling remote clusters the ability to run internal jobs carrying certain security risks. The following list describes measures in places to mitigate those risks.
```

Comment:
Is the list below just a set of mitigations for this one singular security risk? If so, can we be a little more descriptive about the risk? I don't have a lot to add off hand other than enabling malicious actors to pivot into private networks and being able to achieve arbitrary code execution.